File: backend/services/certificate-service/src/main/java/com/lmspilot/certificate/CertificateServiceApplication.java
package com.lmspilot.certificate;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class CertificateServiceApplication {
    public static void main(String[] args) { SpringApplication.run(CertificateServiceApplication.class, args); }
}

File: backend/services/certificate-service/src/main/kotlin/com/lmspilot/certificate/CertificateServiceApplication.kt
package com.lmspilot.certificate

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class CertificateServiceApplication

fun main(args: Array<String>) {
    runApplication<CertificateServiceApplication>(*args)
}
